assets/worker/0700_worker_daemonset.yaml

```diff
@@ -14,15 +14,18 @@ spec:
         app: nfd-worker
     spec:
       tolerations:
-      - operator: "Exists"
-        effect: "NoSchedule"
+        - operator: "Exists"
+          effect: "NoSchedule"
       affinity:
         nodeAffinity:
           requiredDuringSchedulingIgnoredDuringExecution:
             nodeSelectorTerms:
-            - matchExpressions:
-              - key:  node-role.kubernetes.io/master
-                operator: DoesNotExist
+              - matchExpressions:
+                  - key: node-role.kubernetes.io/master
+                    operator: DoesNotExist
+              - matchExpressions:
+                  - key: node-role.kubernetes.io/node
```

Comment:
```suggestion
                  - key: node-role.kubernetes.io/worker
```

Reply:
this is a packported patch from upstream, we need to hot-fix when backporting since in OpenShift we use the term `worker` and not `node`

Reply:
Are you saying this patch needs to be fixed to use `node-role.kubernetes.io/worker` before it is merged?

I note that the 4.7 fix wasn't actually tested on a 3 node cluster: https://bugzilla.redhat.com/show_bug.cgi?id=1897346#c1

Reply:
hey @markmc yes, it is a question, since this is a bot cherry-pick, how can I request the change for `node-role.kubernetes.io/worker` in this PR, can you do that?

Also QA told me has  been tested. maybe need to update the BZ

Reply:
I don't know if you can edit a PR created by the bot, but it's pretty easy for you to close this PR and manually create a PR without the bot's help

Reply:
a PR against  the branch right?

Reply:
@ArangoGutierrez yes just do a PR against release-4.6 close this one.

Reply:
Done  art https://github.com/openshift/cluster-nfd-operator/pull/120
@zvonkok @markmc